extensions.md

```diff
@@ -165,3 +165,24 @@ clangd allows clients to use UTF-8 offsets instead. This allows clients that alw
   - servers that support both UTF-8 and UTF-16 should check whether the client capabilities mentions `"utf-8"` as supported, and use it if so. The selected encoding should be reported in the `InitializeResponse`.
     This allows UTF-8 to be used when the client prefers it.
   - servers that only support UTF-16 can add `offsetEncoding: "utf-16"` or simply not use the extension.
+
+## Code completion scores
+{:.v10}
+
+LSP gives servers limited control over completion display order through the
+`sortText` attribute. Clangd uses several signals such as number of usages to
+ensure "likely" completions are near the top.
```

Comment:
s/likely/the highest quality/g or something similar

Reply:
We refer to "quality" below, and using different ways to say the same thing should aid understanding. I'll add a few more words here though.